examples/show_btrack_widget.py

```diff
@@ -0,0 +1,32 @@
+"""
+Load and show sample data
+=========================
+This example:
+- loads some sample data
+- adds the data to a napari viewer
+- loads the arboretum plugin
+- opens the napari viewer
+"""
+from btrack import datasets
+import napari
+
+viewer = napari.Viewer()
+
+_, btrack_widget = viewer.window.add_plugin_dock_widget(
+    plugin_name="napari-btrack", widget_name="Track"
+)
+
+# populate with example config and data
+cell_config = datasets.cell_config()
+btrack_widget.config_file_path.value = cell_config
+
+
+segmentation = datasets.example_segmentation()
+viewer._add_layer_from_data(segmentation)
```

Comment:
I'm always very wary about using private API from external packages in case it breaks - is there another method from https://napari.org/api/napari.Viewer.html that can be used here to load the data here?

Reply:
`viewer.add_image(...)`?

Reply:
Yep, `viewer.add_image()` does the trick. Fixed.